### ISSUE TITLE
esp_bt_controller_deinit causes wifi disconnect (IDFGH-7326)

### ISSUE THREAD

**Issue by mmyer** [espressif/esp-idf#8913]
## Environment

Module:  [ESP32C3-MINI-1]
IDF version : [4.4.1]

## Problem Description

Wi-Fi disconnect immediately when use esp_bt_controller_deinit to disable BLE.
 
I found this two line in esp_bt_controller_deinit 

```C
    /* Fix the issue caused by the power off the bt power domain.
     * This issue is only on ESP32C3.
     */
    phy_init_flag();

    esp_bt_power_domain_off();
```
esp_bt_power_domain_off seem to turn off wifi
```C
static inline void esp_bt_power_domain_off(void)
{
    // Bluetooth module power down
    SET_PERI_REG_MASK(RTC_CNTL_DIG_ISO_REG, RTC_CNTL_BT_FORCE_ISO);
    SET_PERI_REG_MASK(RTC_CNTL_DIG_PWC_REG, RTC_CNTL_BT_FORCE_PD);
    esp_wifi_bt_power_domain_off();
}
```
After i comment the code before, my problem is solved.
But i don't know if it will cause other problems.
So how do I solve this problem?

**Comment by wmy-espressif** [comment]
Hi @mmyer , thank you for your report and analysis of this issue. This issue is a know issue for us, and it was resolved after v4.4.1 is released. On branch release/v4.4, commit `v4.4.1-74-g7bf3277692` fixes this issue, please use that commit to have a try.

This issue is also reported by other developers, please refer to https://github.com/espressif/esp-idf/issues/8481 . Sorry for the inconvenience.

**Comment by Alvin1Zhang** [comment]
Thanks for reporting, feel free to reopen.
